### DISCUSSION TITLE
How to change <Main /> in next/document to <main /> instead of <div />?

### DISCUSSION THREAD

**Discussion by Pszz** [vercel/next.js#65779] [Help] [answered]
### Summary

**I wish it was more structured and semantic.**

Rendering results

- Now
```html
<div id="__next">
   <header>Website header</header>
   <header>Website content</header>
   <footer>Website header</footer>
</div>
```

- Expected expectations
```html
<main id="__next">
   <header>Website header</header>
   <div>Website content</div>
   <footer>Website footer</footer>
</main>
```

**Comment by icyJoseph** [comment, marked as answer]
Hi,

Yeah I don't think this is possible in Pages Router, https://github.com/vercel/next.js/blob/0b261f0919c82c8b6f1ff08c10657ba9f57123ba/packages/next/src/server/render.tsx#L1215, it is part of a trend that existed back in the day, to mount your app at a div, often called root, etc.

Semantically, it shouldn't matter, though, your main tag can still live within the `__next` root div.

Alternatively, App Router, gives you 100% control over this, so you might want to consider that option.
